src/main/java/org/openlmis/stockmanagement/service/CalculatedStockOnHandService.java

```diff
@@ -92,6 +92,29 @@ public List<StockCard> getStockCardsWithStockOnHand(
     return stockCards;
   }
 
+  /**
+   * Returns list of stock cards with fetched Stock on Hand values.
+   *
+   * @param programId  program id to find stock cards
+   * @param facilityId facility id to find stock cards
+   * @param asOfDate   date used to get latest stock on hand before or equal specific date
+   * @param orderableIds  orderable ids to find stock card
+   * @param lotCodeIds  lot code ids to find stock card
+   * @return List of stock cards with SOH values, empty list if no stock cards were found.
+   */
+  public List<StockCard> getStockCardsWithStockOnHand(
```

Comment:
there should be only one method getstockcardswithstockonhand with logic, other methods should use this one